packages/mediapipe-task-text/lib/src/tasks/text_classification/containers/text_classifier_result.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:mediapipe_core/mediapipe_core.dart';
+import 'package:mediapipe_text/src/mediapipe_text_bindings.dart' as bindings;
+
+class TextClassifierResult {
+  const TextClassifierResult({
+    required this.classifications,
+    this.timestamp,
+  });
+
+  // The classification results for each head of the model.
+  final List<Classifications> classifications;
+
+  // The optional timestamp (as a Duration) of the start of the chunk of data
```

Comment:
Thrice `/` and breadcrumbs if public 😉 

```suggestion
  /// The optional timestamp (as a [Duration]) of the start of the chunk of data
```